src/rules/order.js

```diff
@@ -244,14 +244,35 @@ function getSorter(ascending) {
   const multiplier = ascending ? 1 : -1;
 
   return function importsSorter(importA, importB) {
-    let result;
-
-    if (importA < importB) {
-      result = -1;
-    } else if (importA > importB) {
-      result = 1;
+    let result = 0;
+
+    if (!importA.includes('/') && !importB.includes('/')) {
+      if (importA < importB) {
+        result = -1;
+      } else if (importA > importB) {
+        result = 1;
+      } else {
+        result = 0;
+      }
     } else {
```

Comment:
is there a reason we can't unconditionally split on slash, and iterate the array?